packages/react-components/react-nav-preview/stories/NavItem/NavItemDefault.stories.tsx

```diff
@@ -1,4 +1,60 @@
 import * as React from 'react';
-import { NavItem, NavItemProps } from '@fluentui/react-nav-preview';
+import { Nav, NavItem, NavItemProps } from '@fluentui/react-nav-preview';
+import { Folder20Regular, Folder20Filled } from '@fluentui/react-icons';
 
-export const Default = (props: Partial<NavItemProps>) => <NavItem value={'someValue'} {...props} />;
+import { makeStyles, tokens } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    maxWidth: '260px',
+    rowGap: '60px',
+    paddingTop: '48px',
+    paddingBottom: '48px',
+    paddingLeft: '48px',
+    paddingRight: '48px',
```

Comment:
could this be changed to:
```suggestion
    flexDirection: 'column',
    alignContent: 'center',
    maxWidth: '260px',
    rowGap: '60px'
```

Reply:
yeah, we wanted to keep the padding to simulate the back plate provided by the nav drawer.